mythril/laser/smt/bitvec.py

```diff
@@ -233,7 +572,6 @@ def Concat(*args: Union[BitVec, List[BitVec]]) -> BitVec:
     :param args:
     :return:
     """
-
     # The following statement is used if a list is provided as an argument to concat
     if len(args) == 1 and isinstance(args[0], list):
         bvs = args[0] # type: List[BitVec]
```

Comment:
This variable isn't just a List of BitVecs right? It also contains bitvecfunc

Reply:
A BitVecFunc is a subclass of a BitVec.